packages/storybook-a11y-test/README.md

```diff
@@ -2,65 +2,153 @@
 
 ## Installation
 
+Add this package to your project’s development dependencies.
+
 ```bash
-$ yarn add @shopify/storybook-a11y-test
+$ yarn add @shopify/storybook-a11y-test --dev
+```
+
+This assumes you’ve installed and set up the [Storybook accessibility addon](https://www.npmjs.com/package/@storybook/addon-a11y).
+
+In your project’s `package.json`, add the `build-storybook` and `storybook-a11y-test` scripts:
+
+```json5
+// package.json
+"scripts": {
+  "build-storybook": "build-storybook --static-dir=.storybook/public --output-dir=build/storybook/static",
+  "storybook-a11y-test": "node ./scripts/storybook-a11y-test.js"
+},
+```
+
+### CI steps
+
+Your CI steps should include:
+
+1. Building Storybook (`yarn run build-storybook`)
+2. Running the script (`yarn run storybook-a11y-test`)
+
+For example:
+
+```yaml
+steps:
+  - label: ':storybook: Build Storybook and run accessibility tests'
+    run:
+      - yarn install
+      - yarn run build-storybook
+      - yarn run storybook-a11y-test
 ```
 
+For optimal test performance, break the build and accessibility testing steps into two separate steps, as illustrated in [🔒 this example](https://github.com/Shopify/web/blob/03475cd0918c0e18989c1974d7007c1bd912e054/.shopify-build/shared/steps.yml#L154-L184) (only visible to Shopify employees).
+
 ## Usage
 
-Make sure you have built your storybook instance and there is an `iframe.html` file that you can point the test towards.
+Make sure you have built your Storybook instance and there is an `iframe.html` file that you can point the test towards.
 
 ```js
-const {testPages, getCurrentStoryIds} = require('./built-a11y-package');
+const {testPages, getCurrentStoryIds} = require('@shopify/storybook-a11y-test');
 
 (async () => {
+  // Custom URL or file path to Storybook’s iframe
   const iframePath = `file://${__dirname}/../build/storybook/static/iframe.html`;
-  const timeout = 1000;
-  const storyIds = await getCurrentStoryIds({iframePath, skippedStoryIds: []});
-  const results = await testPages({iframePath, storyIds, timeout});
+
+  // Grab all Story IDs
+  const storyIds = await getCurrentStoryIds({
+    iframePath,
+
+    // Optional, IDs of stories that shouldn’t be tested (for example: playgrounds)
+    skippedStoryIds: [],
+  });
+
+  // Run tests on all stories in `storyIds`
+  const results = await testPages({
+    iframePath,
+    storyIds,
+
+    // Optional: maximum time in milliseconds to wait for the browser instance to start.
+    // Defaults to 30000 (30 seconds). Pass 0 to disable timeout.
+    timeout: 30000,
+  });
 
   if (results.length) {
-    console.error(`‼️  Test failures found`);
+    console.error(`‼️  Accessibility violations found`);
     console.log(results.join('\n'));
     process.exit(1);
   } else {
-    console.log('🧚‍♀️ Accessibility is all g');
+    console.log('🧚  Accessibility tests passed');
   }
 })();
 ```
 
+### Ignoring specific violations in a Story
```

Comment:
Could we add something here about only doing this if there are false positives, not to get around an existing accessibility problem being flagged?